pandas/core/groupby.py

```diff
@@ -1736,13 +1730,14 @@ class BaseGrouper(object):
     """
 
     def __init__(self, axis, groupings, sort=True, group_keys=True,
-                 mutated=False):
+                 mutated=False, indexer=None):
         self._filter_empty_groups = self.compressed = len(groupings) != 1
```

Comment:
can you add a doc-string explaining params (I know you just added 1 but good time)

Reply:
I'm sorry I don't really understand all parameters, I've added those I know😂